lib/routes/nautiljon/manga-releases.ts

```diff
@@ -0,0 +1,70 @@
+import { load } from 'cheerio';
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import puppeteer from '@/utils/puppeteer';
+import logger from '@/utils/logger';
+
+const host = 'https://www.nautiljon.com';
+export const route: Route = {
+    path: '/releases/manga',
+    categories: ['reading'],
+    example: '/releases/manga',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['nautiljon.com/'],
+        },
+    ],
+    name: 'Nautiljon France manga releases',
+    maintainers: ['Fafnor'],
+    handler,
+    url: 'nautiljon.com',
+};
+
+async function handler() {
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+    await page.setRequestInterception(true);
+    page.on('request', (request) => {
+        request.resourceType() === 'document' ? request.continue() : request.abort();
+    });
+
+    const link = `${host}/planning/manga/`;
+    logger.http(`Requesting ${link}`);
+    await page.goto(link, {
+        waitUntil: 'domcontentloaded',
+    });
+    const response = await page.content();
+    page.close();
```

Comment:
Unnecessary use of puppeteer. ofetch works fine if [RSSHub's user-agent](https://github.com/DIYgod/RSSHub/blob/a70f3e4c347a931cc8e3c9dde826c4344b8199d6/lib/config.ts#L23) is used.